libs/community/langchain_community/vectorstores/neo4j_vector.py

```diff
@@ -1030,9 +1033,12 @@ def similarity_search_with_score_by_vector(
             base_index_query = parallel_query + (
                 f"MATCH (n:`{self.node_label}`) WHERE "
                 f"n.`{self.embedding_node_property}` IS NOT NULL AND "
-                f"size(n.`{self.embedding_node_property}`) = "
```

Comment:
The embedding dimension comes from embedding model and not index definition, so this is not needed:

https://github.com/langchain-ai/langchain/blob/2ddf5b27f0e6f3c4d4a670f28a0e6f0860a6c575/libs/community/langchain_community/vectorstores/neo4j_vector.py#L576

Reply:
Removed, thank you!